SmartEVSE2.X/modbus.c

```diff
@@ -457,9 +457,9 @@ signed long receiveMeasurement(unsigned char *buf, unsigned char pos, unsigned c
     if (IsDouble) {
         combineBytes(&dCombined, buf, pos, Endianness);
         if (Divisor >= 0) {
-            lCombined = dCombined / (signed long)pow10[Divisor];
```

Comment:
2x typecast added as compiler throws warning _(356) implicit conversion of float to integer when executed_ at warning level -2

Reply:
My XC8 compiler (v2.10) doesn't complain about this, but thanks for the fix.